main.go

```diff
@@ -20,35 +20,29 @@ import (
 	"time"
 
 	"github.com/layer5io/meshery-adapter-library/adapter"
-	"github.com/layer5io/meshery-adapter-library/config/provider"
+	configprovider "github.com/layer5io/meshery-adapter-library/config/provider"
 	"github.com/layer5io/meshery-consul/internal/operations"
 
 	"github.com/layer5io/meshery-adapter-library/api/grpc"
 	"github.com/layer5io/meshery-consul/consul"
 	"github.com/layer5io/meshery-consul/internal/config"
 	"github.com/layer5io/meshkit/logger"
-	"github.com/layer5io/meshkit/utils"
 )
 
 const (
 	serviceName = "consul-adapter"
 )
 
-var (
-	kubeConfigPath = fmt.Sprintf("%s/.kube/meshery.config", utils.GetHome())
-)
-
 func main() {
 	log, err := logger.New(serviceName, logger.Options{Format: logger.JsonLogFormat, DebugLevel: false})
 	if err != nil {
 		fmt.Println("Logger Init Failed", err.Error())
 		os.Exit(1)
 	}
 
-	cfg, err := config.New(provider.Options{
+	cfg, err := config.New(configprovider.Options{
 		ServerConfig:   config.ServerDefaults,
 		MeshSpec:       config.MeshSpecDefaults,
-		MeshInstance:   config.MeshInstanceDefaults,
```

Comment:
Is MeshInstance no longer supported in MeshKit?

Reply:
No, this config entry has been removed (from meshery-adapter-library). 

I think it would be good if the design (in general, but also the related data models) was reviewed and documented before migrating several adapters. 
@leecalcote @Aisuko @kumarabd what do you think?

Reply:
Yep, let's do documented first.